website-next/tailwind-preset.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
     fontFamily: {
       headline: ['RecklessNeue-Book', 'serif'],
       menu: ['RecklessNeue-Book', 'serif'],
-      body: ['Inter', 'serif'],
+      body: ['Inter', 'sans-serif'],
```

Comment:
ooh, this was a leftover from debugging – I wanted to see when the font fails loading, that's why I set it to serif so it's very glaring difference :)

Reply:
...and let me tell you, it worked :D